prefect-workflows.md

```diff
@@ -0,0 +1,103 @@
+Prefect: Production Workflows
+============================
+
+Who am I?
+---------
+
+I am [Chris White](http://github.com/cicdw); after getting my Ph.D. in Math at 
+UT-Austin, I transitioned to data science with an emphasis on data science tooling.  I now work as the Tech Lead 
+at [Prefect](https://www.prefect.io), a company building the next generation of workflow automation platforms for data engineers and data
+scientists.  In this role, I am the core developer of our [open source engine](https://github.com/PrefectHQ/prefect)
+which allows users to build, schedule and execute robust workflows.
+
+The Problem I'm trying to solve
+-------------------------------
+
+Most teams are responsible for maintaining production workflows that
+are critical to the team's mission. Historically these workflows consisted
+largely of batch ETL jobs, but more recently include things such as
+deploying parametrized machine learning models, ad-hoc reporting, and
+handling event-driven processes.
+
+Typically this means developers need a workflow system which can do things such as:
+- retry failed tasks
+- schedule jobs to run automatically
+- log detailed progress (and history) of the workflow
+- provide a dashboard / UI for inspecting system health
+- provide notification hooks for when things go wrong
+
+among many other things.  We at Prefect like to think of a workflow system as
+a technical insurance policy - you shouldn't really notice it much when 
+things are going well, but it should be maximally useful when things go wrong.
+
+Prefect's goal is to build the next generation workflow system.  [Old systems
+are limited by their model of workflows](https://medium.com/the-prefect-blog/why-not-airflow-4cfa423299c4) as slow-moving, regularly scheduled,
```

Comment:
Should we call out Airflow specifically here?  Maybe not exactly *here*, but somewhere, just to put people in the right frame of mind.

Reply:
Yea, good idea - I'll call out Airflow and Luigi here.